tools/hooks/restrict_levelbuilder_changes.rb

```diff
@@ -4,7 +4,9 @@
 BLOCKS_DIR = File.expand_path(REPO_DIR + '/dashboard/config/blocks', __FILE__).freeze
 SHARED_FUNCTIONS_DIR = File.expand_path(REPO_DIR + '/dashboard/config/shared_functions', __FILE__).freeze
 LIBRARIES_DIR = File.expand_path(REPO_DIR + '/dashboard/config/libraries', __FILE__).freeze
-LEVELS_DIR = File.expand_path(REPO_DIR + '/dashboard/config/scripts', __FILE__).freeze
+# As written, SCRIPTS_DIR also covers dashboard/config/scripts_json
+SCRIPTS_DIR = File.expand_path(REPO_DIR + '/dashboard/config/scripts', __FILE__).freeze
```

Comment:
I assume we checked/it doesn't matter that this rename from LEVELS_DIR to SCRIPTS_DIR doesn't have an impact anywhere else?

Reply:
Yep! The string `LEVELS_DIR` appears in three files (including this one) but all of them define the constant within the file.

![Screenshot 2023-08-15 at 10 13 13 AM](https://github.com/code-dot-org/code-dot-org/assets/46464143/e1eb9208-497b-43aa-8480-b62b29aa8f3c)